metrics/ci_integrations/docs/02_ci_integration_user_guide.md

```diff
@@ -148,6 +148,18 @@ For example:
 ci_integrations sync --config-file="path/to/config_file.yaml"
 ```
 
+#### Build coverage importing
+
+By default, build coverage data is fetching along with builds. 
+
+If the given `Access Token` has no permission to read build artifacts in CI the `sync` command will fail. To skip fetching the coverage step, you should use the `sync` command with the `--skip-coverage` flag.
+
+For example:
+
+```bash
+ci_integrations sync --config-file="path/to/config_file.yaml" --skip-coverage
+```
+
```

Comment:
```suggestion
_**Note**: All builds synchronized with the `--skip-coverage` flag won't contain any coverage information. This may affect the coverage metric available on the Metrics Web Application._

```